### PR TITLE
Microrts Migration

### PR DESCRIPTION
**Pull request by vwxyzjn**
This PR fully migrates to the microrts master

cc @DennisSoemers

### CODE REVIEW

File: experiments/league.py [hunk 1]
```diff
@@ -91,9 +91,7 @@ def parse_args():
     from ppo_gridnet import Agent, MicroRTSStatsRecorder
 
     from gym_microrts.envs.vec_env import MicroRTSBotVecEnv
-    from gym_microrts.envs.vec_env import (
-        MicroRTSGridModeSharedMemVecEnv as MicroRTSGridModeVecEnv,
-    )
+    from gym_microrts.envs.vec_env import MicroRTSGridModeVecEnv
```
Comment (vwxyzjn) [inline experiments/league.py:94]: We will just use `MicroRTSGridModeVecEnv ` for the new migration for ease of maintenance

File: experiments/league.py [hunk 2]
```diff
@@ -202,6 +201,7 @@ def __init__(self, partial_obs: bool, match_up=None, map_path="maps/16x16/basesW
                 render_theme=2,
                 map_paths=[map_path],
                 reward_weight=np.array([10.0, 1.0, 1.0, 0.2, 1.0, 4.0]),
+                autobuild=False,
```
Comment (vwxyzjn) [inline experiments/league.py:204]: disable wasteful autobuild during eval.

File: gym_microrts/envs/vec_env.py [hunk 3]
```diff
@@ -516,7 +523,7 @@ def step_wait(self):
         reward, done = np.array(responses.reward), np.array(responses.done)
         infos = [{"raw_rewards": item} for item in reward]
         # check if it is in evaluation, if not, then change maps
-        if len(self.cycle_maps) > 0:
+        if len(self.cycle_maps) > 1:
```
Comment (vwxyzjn) [inline gym_microrts/envs/vec_env.py:526]: We should only swap to different training maps if there are more maps.

**Comment by DennisSoemers** [comment]
Looks good to me. Once this has been merged (plus, importantly, first the PR in the Java repo, since this PR in the python repo will kind of depend on that one due to the submodule change), I'll be happy to test and see if everything's fine.

**Pull request merged**